examples/deadline/Local-Zone/ts/bin/config.ts

```diff
@@ -38,14 +38,14 @@ class AppConfig {
    * "10.1.16"
    * @default 10.3.2 is used, to match the worker AMI ID provided below
```

Comment:
```suggestion
   * @default 10.4.0 is used, to match the worker AMI ID provided below
```
Nitpick: looks like the "default" version in this comment should be updated from 10.3.2 to 10.4.0